### PR TITLE
Fix alert issue in shipment page

### PR DESCRIPTION
**Pull request by TanmayJain92**
## Purpose
Alert issue in shipment page

### CODE REVIEW

File: src/layout/TopBar/components/AlertNotifications.js [hunk 1]
```diff
@@ -57,10 +60,15 @@ const AlertNotifications = ({
...
-    queryClient.invalidateQueries({ queryKey: ['allGateways'] });
-    queryClient.invalidateQueries({ queryKey: ['sensorAlerts'] });
-    queryClient.invalidateQueries({ queryKey: ['sensorReports'] });
+    if (oauthService.hasValidAccessToken()) {
+      queryClient.invalidateQueries({ queryKey: ['shipments'] });
+      queryClient.invalidateQueries({ queryKey: ['allGateways'] });
+      queryClient.invalidateQueries({ queryKey: ['sensorAlerts'] });
+      queryClient.invalidateQueries({ queryKey: ['sensorReports'] });
+      setRedirectToLogin(false);
+    } else {
```
Comment (patelradhika) [inline src/layout/TopBar/components/AlertNotifications.js:69]: Add oauthService.logout() in else part

**Review by patelradhika — CHANGES_REQUESTED** [review]
Make suggested changes

File: src/layout/TopBar/components/AlertNotifications.js (hunk 1 above)
Comment (TanmayJain92) [reply src/layout/TopBar/components/AlertNotifications.js:69]: done

**Review by patelradhika — APPROVED** [review]
Looks good

**Pull request merged**